libr/include/r_types.h

```diff
@@ -426,6 +426,8 @@ static inline void *r_new_copy(int size, void *data) {
 #define HHXFMT  "hhx"
 #endif
 
+#define PFMTDPTR "td"
```

Comment:
didnt knew about this format expression. it is also supported by msvc... i assume other libcs will handle that too.

according to stackoverflow:

`Visual Studio 2015 supports both %zu and %td (not properly documented on MSDN, but both test out OK). – vladr Dec 18 '17 at 21:34`

so we should be fine with that i guess